app/models/profile.rb

```diff
@@ -15,6 +15,7 @@ class Profile < ApplicationRecord
     encryption_error: 2,
     verification_pending: 3,
     verification_cancelled: 4,
+    in_person_verification_pending: 5,
```

Comment:
Since the enums are plain ints in the Db, what about renaming 3 to `gpo_verification_pending`?

Reply:
> Since the enums are plain ints in the Db, what about renaming 3 to `gpo_verification_pending`?

Yeah, I think that makes sense. Along the same lines, in an earlier iteration I had also renamed the `UserDecorator#pending_profile_requires_verification?` method to `pending_profile_requires_gpo_verification?`. I have a feeling there's a lot of different references to this "pending" state, so I may want to be a bit conservative with those updates for now. I think renaming the deactivation reason may be a good start for now at least.

Reply:
Also, it may be worth noting that, in practice, "verification pending" is used exclusively to mean GPO verification, the process of creating and activating a profile does seem to treat it as a bit more generic, i.e. any new profile is pending by default until activated ([see `Idv::ProfileMaker#save_profile`](https://github.com/18F/identity-idp/blob/b25790b34d3c542f00849808dc37e8d0a7ec64e7/app/services/idv/profile_maker.rb#L11-L20)).

Reply:
I didn't realize that profiles start as pending like that. That makes me think maybe we would want to keep a single pending enum and maybe add a separate enum column for what it's waiting on (gpo, ipp, etc), unsure if deactivation reason is a super good fit for that? But it is already there.

Reply:
That's what I was getting at with the second paragraph of my previous comment. It seems like we already have the two-field "pending" + "what it's waiting on" via `active: false` + `deactivation_reason:`. Currently, we initialize an `active: false` profile with `deactivation_reason: :verification_pending`, but I'm not sure there's a reason we have to?

Reply:
I pushed what I had in mind in 3dec5e2c2 . Also worth noting that with the logic flow for initializing a profile, the profile would immediately be either activated or assigned `deactivation_reason: :gpo_verification_pending` or `deactivation_reason: :in_person_verification_pending` depending on the route the user had selected during the proofing attempt.

https://github.com/18F/identity-idp/blob/b25790b34d3c542f00849808dc37e8d0a7ec64e7/app/controllers/idv/review_controller.rb#L89-L91

Because of this, we could even consider changing `save_profile` to accept `deactivation_reason` as an optional argument and assign it at creation, which could have the added advantage of avoiding a second database update.

Reply:
Yeah that seems the clearest overall. And yes also updating `save_profile` to save a round trip seems worthwhile as well, maybe for a follow up PR if this one is getting bogged down

Reply:
I'm going to merge this and follow-on with a few other improvements (more exhaustive specs, etc). I may do it there or in a follow-on-follow-on.

Reply:
> I'm going to merge this and follow-on with a few other improvements (more exhaustive specs, etc). I may do it there or in a follow-on-follow-on.

For posterity: #6680, #6681